sources/packages/api/src/services/application/application.service.ts

```diff
@@ -92,11 +100,12 @@ export class ApplicationService extends RecordDataModelService<Application> {
    */
   async submitApplication(
     applicationId: number,
+    auditUserId: number,
     studentId: number,
     programYearId: number,
     applicationData: any,
```

Comment:
we can change the type any to ApplicationData

Reply:
@ann-aot I changed the service layer to use the ApplicationData as we talked but when I do it on the DTO it impact too many places outside the PR, so I will not do the change on DTO right now.